frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -780,13 +780,26 @@ public static function apiList(\OmegaUp\Request $r) {
             ['promotion', 'demotion']
         );
 
-        return \OmegaUp\DAO\QualityNominations::getNominations(
+        $response = \OmegaUp\DAO\QualityNominations::getNominations(
             /* nominator */ null,
             /* assignee */ null,
             $offset,
             $rowCount,
             $types
         );
+
+        $pagerItems = \OmegaUp\Pager::paginate(
+            $response['totalRows'],
+            $rowCount,
+            $offset,
+            5,
+            []
```

Comment:
puedes ponerle el comentario de qué son estas variables? 

```suggestion
            /*$adjacent=*/5,
            /*$params=*/[]
```

igual en los otros lugares.